crypto/thread/arch/thread_win.c

```diff
@@ -32,6 +32,17 @@ static unsigned __stdcall thread_start_thunk(LPVOID vthread)
     return 0;
 }
 
+// To support Embarcadero's clang based compilers
```

Comment:
There also appears to be an issue with the use of the ' character in the comment, see
https://github.com/openssl/openssl/actions/runs/6507478087/job/17674975217?pr=22361